native/src/main/java/bblfsh/bash/Main.java

```diff
@@ -0,0 +1,18 @@
+package bblfsh.bash;
+
+public class Main {
+
+    public static void main(String args[]) {
+        final RequestReader reader = new RequestReader(System.in);
```

Comment:
As the ctor of `RequestReader` and `ResponseWriter` is public, maybe I should made `Main.defaultObjectMapper` public too, what do you think?